CMakeLists.txt

```diff
@@ -1,10 +1,8 @@
-cmake_minimum_required(VERSION 2.6.4)
+cmake_minimum_required(VERSION 3.1)
 
 option(WITH_TESTS           "Build tests."       OFF)
 option(WITH_EXAMPLE         "Build example."     OFF)
 option(WITH_STATIC          "Build static libs." ON)
-option(WITH_CPP11           "Build with C++11."  ON)
-option(WITH_CPP14           "Build with C++14."  OFF)
```

Comment:
Oh ok that makes sense. I'm still new to both cmake and Travis, so any help you offer is greatly appreciated!